addons/semicolon/semicolon.js

```diff
@@ -0,0 +1,4 @@
+var semicolon = document.createElement("p");
```

Comment:
You should likely readd the semicolon if the user goes outside project editor mode. Also, you should likely not add the semicolon if the initial page is the editor.

Reply:
It's going to be a hidden easter egg, it may as well be everywhere.